### DISCUSSION TITLE
playlist support

### DISCUSSION THREAD

**Discussion by gvmii** [Tandashi/YTSync-Plugin#45] [Ideas]
playlist support

**Comment by Tandashi** [comment]
Could you maybe try to elaborate a bit more what you mean by `playlist support`?

Do you mean to make it possible to add playlists to the queue? Add the Queue as a playlist to your YouTube account?

**Reply by gvmii** [reply]
I wasn't really thinking when I made this Idea, I apologize first. 

Maybe it's already implemented, and I just didn't manage to do it correctly, but my friends and I like to listen to playlists together (We like using different quality options, because internet stuff) so having a way to add entire playlists to the queue would come very handy.

**Reply by Tandashi** [reply]
No, that is currently not implemented, so you and your friends haven't missed anything. I could see this being implemented in the near future (not sure when exactly that will be, maybe during the weekend when I find time but don't want to promise anything).

Was thinking a button on a playlist (as seen in the attached pictures) would do the trick.
If that is what you imagined, would love to get some feedback on that.
Or if you had put it somewhere else entirely.

![image](https://user-images.githubusercontent.com/18377875/150442782-83293351-88a7-4c81-9c53-16934cdf1060.png)
![image](https://user-images.githubusercontent.com/18377875/150442759-a0e6aea0-5c99-4729-99cb-0d2605cd9f88.png)

**Reply by gvmii** [reply]
This looks really good! Really intuitive as well. I also think it should be here in the Playlist page 
![image](https://user-images.githubusercontent.com/68835899/150619567-1f6d2850-0fed-4fa9-9262-cdf3ca9c8dbc.png)

**Reply by Tandashi** [reply]
If you can't wait for the full release there is a pre-release version with playlist support now (`2.1.0-pre1`) if you have Tampermonkey.

https://github.com/Tandashi/YTSync-Plugin/releases/tag/v2.1.0-pre1

Or you could build it from source and add it as an unpacked extension.

Note: Also works with private playlists like (liked videos, watch later, private custom mixes, ...)

**Comment by Tandashi** [comment]
Closing the discussion since this feature will be part of the upcoming `v2.1.0` release
